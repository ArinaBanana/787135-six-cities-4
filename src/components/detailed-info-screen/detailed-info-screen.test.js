import React from "react";
import renderer from "react-test-renderer";
import DetailedInfoScreen from "./detailed-info-screen.jsx";

const places = [
  {
    id: 7,
    title: `Foo bar`,
    price: 45,
    img: `path`,
    type: `apartment`,
    rating: `20%`,
    isPremium: false,
    isBookmark: true,
    color: `path`,
    coordinates: [52.3809553943508, 4.939309666406198]
  }
];

const reviews = [
  {
    id: 1,
    username: `Jon`,
    rating: `70%`,
    message: `message`,
    date: `date`,
  }
];

it(`Should render Detailed Info Screen`, () => {
  const tree = renderer.create(<DetailedInfoScreen
    place={places[0]}
    reviews={reviews}
    nearPlaces={places} />, {createNodeMock: () => {
    return document.createElement(`div`);
  }}).toJSON();

  expect(tree).toMatchSnapshot();
});
