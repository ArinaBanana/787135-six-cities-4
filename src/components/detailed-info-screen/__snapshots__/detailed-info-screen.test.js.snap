// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Detailed Info Screen 1`] = `null`;
